src/types/big-integer.ts

```diff
@@ -9,13 +9,29 @@ export const types = {
 	},
 };
 
-export type Types = TypeUtils.TsTypes<number, number>;
-type DbWriteType = number;
+export type Types = TypeUtils.TsTypes<bigint, bigint>;
+type DbWriteType = bigint;
+
+export const fetchProcessing: TypeUtils.FetchProcessing<Types['Read']> = (
+	data,
+) => {
+	if (data == null) {
+		return data;
+	}
+
+	if (typeof data === 'bigint') {
+		return data;
+	} else if (typeof data === 'string' || typeof data === 'number') {
+		return BigInt(data);
+	}
+
+	throw new Error('Fetched bigint is not valid: ' + typeof data);
+};
```

Comment:
I'd prefer to have it in the else branch. We have different approachers in sbvr-types but ending in the `else{ }` is commonly used. In addition I'm wondering if this is needed at all as BigInt is save to be called with any other type and will throw on its own.  I understand that we want to raise a custom error when the database stores an unexpected value, though.
```suggestion
	} else {
		throw new Error('Fetched bigint is not valid: ' + typeof data);
	}
};
```

Reply:
Updated the branching to match that of other types